documentation/staging/content/userguide/introduction/architecture.md

```diff
@@ -163,7 +161,7 @@ The external state approach allows the operator to treat the images as essential
 
 This approach also eliminates the need to manage any state created in a running container, because all of the state that needs to be preserved is written into either the persistent volume or a database backend. The containers and pods are completely throwaway and can be replaced with new containers and pods, as necessary.  This makes handling failures and rolling restarts much simpler because there is no need to preserve any state inside a running container.
 
-When users wish to apply a binary patch to WebLogic Server, it is necessary to create only a single new, patched image.  If desired, any domains that are running may be updated to this new patched image with a rolling restart, because there is no state in the containers.
+When users wish to apply a binary patch to WebLogic Server, it is necessary to create only a single new, patched image.  If desired, any domains that are running may be updated to this new patched image with a rolling restart, because there is no state in the containers. **Note**: This applies _only_ to the Domain on PV or Model in Image with Auxiliary Image [use cases]({{< relref "/userguide/managing-domains/choosing-a-model/" >}}).
```

Comment:
_Any_ domain home source types may need to be patched and rolled - but the write-up now implies that this is impossible.  

Furthermore the existence of ephemeral state isn't that important here and is adding confusion IMO. So instead of going into detail, how about remove "because there is no state in containers", remove the mention of specific domain home source types, and simply have wording something like:

When users wish to apply a binary patch to WebLogic Server, it is necessary to create only a single new, patched image. If desired, any domains that are running may be updated to this new patched image with a rolling restart.  See WebLogic images doc. See life-cycle doc.

Reply:
Done.